commerce_coordinator/apps/lms/views.py

```diff
@@ -4,76 +4,53 @@
 import logging
 from urllib.parse import unquote, urlencode
 
-from django.conf import settings
-from django.http import HttpResponseRedirect, HttpResponseServerError
+from django.http import HttpResponseBadRequest, HttpResponseRedirect
 from edx_rest_framework_extensions.permissions import LoginRedirectIfUnauthenticated
-from rest_framework.parsers import JSONParser
 from rest_framework.status import HTTP_303_SEE_OTHER
 from rest_framework.throttling import UserRateThrottle
 from rest_framework.views import APIView
 
-from ..core.constants import HttpHeadersNames, MediaTypes, QueryParamPrefixes, WaffleFlagNames
-from .filters import OrderCreateRequested
-from .serializers import OrderCreatedSignalInputSerializer
+from commerce_coordinator.apps.lms.filters import PaymentPageRedirectRequested
+
+from ..core.constants import HttpHeadersNames, MediaTypes, QueryParamPrefixes
 
 logger = logging.getLogger(__name__)
 
 
-class OrderCreateView(APIView):
-    """Accept incoming request for creating a basket/order for a user."""
+class PaymentPageRedirectView(APIView):
+    """Accept incoming request for routing users to checkout view."""
     permission_classes = (LoginRedirectIfUnauthenticated,)
     throttle_classes = (UserRateThrottle,)
```

Comment:
Can you double check if the throttle rate settings are the same as legacy ecommerce endpoint?
Specifically I didn't find any default throttle rate settings in CC.
For reference: https://github.com/openedx/ecommerce/blob/master/ecommerce/settings/base.py#L527

Reply:
Can confirm that it's the same

Reply:
I didn't find any default throttle rate setting in the settings files. Can you point me to that?

Reply:
https://github.com/edx/commerce-coordinator/blob/2u/project-replatforming/commerce_coordinator/settings/base.py#L283